plugins/webp-uploads/settings.php

```diff
@@ -180,20 +180,12 @@ function webp_uploads_generate_webp_jpeg_setting_callback(): void {
 		<script>
 			// Listen for clicks on the JPEG output checkbox, enabling/disabling the
 			// picture element checkbox accordingly.
-			document.addEventListener( 'DOMContentLoaded', function() {
-				const jpegCheckbox               = document.getElementById( 'perflab_generate_webp_and_jpeg' );
-				const pictureCheckbox            = document.getElementById( 'webp_uploads_use_picture_element' );
-				const pictureCheckboxLabel       = document.getElementById( 'webp_uploads_use_picture_element_label' );
-				const pictureCheckboxDescription = document.getElementById( 'webp_uploads_use_picture_element_description' );
-
-				function togglePictureCheckbox() {
-					pictureCheckbox.disabled = ! jpegCheckbox.checked;
-					pictureCheckboxLabel.classList.toggle( 'webp-uploads-disabled', ! jpegCheckbox.checked );
-					pictureCheckboxDescription.classList.toggle( 'webp-uploads-disabled', ! jpegCheckbox.checked );
-				}
-
-				jpegCheckbox.addEventListener( 'change', togglePictureCheckbox );
-			});
+			document.getElementById( 'perflab_generate_webp_and_jpeg' ).addEventListener( 'change', function () {
+				document.querySelector( '.webp-uploads-use-picture-element' ).classList.toggle( 'webp-uploads-disabled', ! this.checked );
+				document.getElementById( 'webp_uploads_picture_element_notice' ).hidden = this.checked;
+				document.getElementById( 'webp_uploads_use_picture_element' ).classList.toggle( 'disabled', ! this.checked );
```

Comment:
By using the `disabled` class as opposed to the attribute, then the input remains in the form submission.